prog/vnet/subnet_nexus.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Prog::Vnet::SubnetNexus < Prog::Base
+  subject_is :ps
```

Comment:
This can't work right? Like, it has to be subject_is `:private_subnet` or something that can be CamelCased into a class...right?

Reply:
yes... sorry, that was a horrible miss. I changed it.